core/src/main/java/io/seata/core/serializer/SerializerType.java

```diff
@@ -43,13 +43,6 @@ public enum SerializerType {
      */
     KRYO((byte)0x4),
 
-    /**
-     * The fst.
-     * <p>
-     * Math.pow(2, 3)
-     */
-    FST((byte)0x8),
```

Comment:
Consider whether this enumeration type does not need to be removed or add a line of comments here to prevent future developers from reusing deprecated code when expanding the serialization protocol. However, in situations where some user clients have not been upgraded, code conflicts may arise, leading to unexpected problems.

Reply:
> Consider whether this enumeration type does not need to be removed or add a line of comments here to prevent future developers from reusing deprecated code when expanding the serialization protocol. However, in situations where some user clients have not been upgraded, code conflicts may arise, leading to unexpected problems.
> 
> 考虑一下，是不是这个枚举类型无需移除，或者在这里加一行注释，防止后续的开发者，再扩展序列化协议的时候，复用了废弃的code，但是部分用户客户端还没有升级的情况下，code冲突，出现预期外的问题

Good point. That's a better way to handle it.

Reply:
down